dist/pom.xml

```diff
@@ -94,6 +107,30 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>net.alchim31.maven</groupId>
+        <artifactId>scala-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>update_config</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <launchers>
+            <launcher>
+              <id>update_rapids_config</id>
+              <mainClass>com.nvidia.spark.rapids.RapidsConf</mainClass>
+              <args>
+                <arg>${project.basedir}/../docs/configs.md</arg>
```

Comment:
doc generation had to move because it requires shim layer functions

Reply:
Is that why `spark-sql` was added as a dependency?  I wasn't clear on why that was needed otherwise.

Reply:
yes